backend/app/routers/licenses.py

```diff
@@ -16,24 +21,39 @@ async def save_license(
     license_in: LicenseIn,
     user=Depends(get_current_user),
 ):
+    if license_in.holders == None:
+        license_in = user
     license_db = LicenseDB(**license_in.dict(), creator=user.email)
     await license_db.insert()
     return license_db.dict()
 
 
-@router.get("/{dataset_id}", response_model=LicenseOut)
-async def get_license(dataset_id: str):
-    if (
-        license := await LicenseDB.find_one(
-            LicenseDB.dataset_id == PydanticObjectId(dataset_id)
-        )
-    ) is not None:
+@router.get("/{license_id}", response_model=LicenseOut)
+async def get_license(license_id: str):
+    if (license := await LicenseDB.get(PydanticObjectId(license_id))) is not None:
         return license.dict()
     raise HTTPException(
-        status_code=404, detail=f"License not found for dataset {dataset_id}"
+        status_code=404, detail=f"License not found for id {license_id}"
     )
 
 
+# Endpoint to retrieve standard license options
+@router.get("", response_model=List[LicenseOption])
```

Comment:
@router.get("/standard_licenses/{license_id}", response_model=str)
Would you like to match the URL pattern and change this to @router.get("/standard_licences")?

Reply:
will update the name